quipucords/quipucords-ui
# docs: updates for current development strategy, resources

## What's included

- docs: updates for current development strategy, resources

### Notes

Provides slight adjustments to the existing documentation to be more reflective of the current development strategy.
- basic updates reflective of the move from yarn to npm
- basic git and build strategies

## How to test
1. Read the `README.md`. Confirm basic copy and linking
1. Read the `CONTRIBUTING.md`. Confirm basic copy and linking






## Example

...

## Updates issue/story


ongoing
